### PR TITLE
Proofread some documents

### PR DESCRIPTION
**Pull request by uzluisf**
Changes mostly involve text reflow, whitespace and correcting minor typos.

### CODE REVIEW

File: doc/Language/exceptions.pod6 [hunk 1]
```diff
@@ -7,7 +7,7 @@
 Exceptions in Perl 6 are objects that hold information about errors. An
 error can be, for example, the unexpected receiving of data or a network
 connection no longer available, or a missing file. The information that
-an exception objects store is, for instance, a human-readable message
+an exception object store is, for instance, a human-readable message
```
Comment (JJ) [inline doc/Language/exceptions.pod6]: stores?

File: doc/Language/io-guide.pod6 [hunk 2]
```diff
@@ -68,49 +71,51 @@ can handle them, so there's no need to convert them.
...
+these two routines are for you.
 
 =for code
 "my-file.txt".IO.spurt: "I ♥ Perl!";
 
 The code above creates a file named C<my-file.txt> in the current directory
 and then writes text C<I ♥ Perl!> into it. If Perl 6 is your first language,
-celebrate your accomplishment! Try to open the file you created with some other
-program to verify what you wrote with your program. If you already know
+celebrate your accomplishment! Try to open the file you created with some
```
Comment (JJ) [inline doc/Language/io-guide.pod6]: *a* text editor?

File: doc/Language/exceptions.pod6 (hunk 1 above)
Comment (uzluisf) [reply doc/Language/exceptions.pod6]: Thanks!

File: doc/Language/io-guide.pod6 (hunk 2 above)
Comment (uzluisf) [reply doc/Language/io-guide.pod6]: Just to be sure, do you mean "some text editor" -> "a text editor"?

Comment (JJ) [reply doc/Language/io-guide.pod6]: Yep.

**Pull request merged**